IdentityCore/src/controllers/broker/mac/MSIDXpcSilentTokenRequestController.m

```diff
@@ -28,20 +28,44 @@
 #import "MSIDLogger+Internal.h"
 #import "MSIDXpcProviderCache.h"
 
+@interface MSIDXpcSilentTokenRequestController ()
+
+// shouldSkipXpcRequest is used when the fallback is not needed for non ASAuthorizationErrorDomain or MSIDErrorSSOExtensionUnexpectedError
+@property (nonatomic) BOOL shouldSkipAcquireToken;
```

Comment:
The comment says shouldSkipXpcRequest but property is named shouldSkipAcquireToken. Can you rename property to shouldSkipCallingXpc? shouldSkipAcquireToken is confusing